File: Python/tests/test_cra_linear.py
import pytest

from hbayesdm.models import cra_linear


def test_cra_linear():
    _ = cra_linear(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_cra_linear.R
context("Test cra_linear")
library(hBayesDM)

test_that("Test cra_linear", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(cra_linear(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
